Source/WebCore/platform/cocoa/WebAVPlayerLayer.mm

```diff
@@ -127,14 +127,14 @@ - (void)setPresentationModel:(VideoPresentationModel*)presentationModel
         model->removeClient(*_presentationModelClient);
 
     _presentationModel = presentationModel;
+#if !RELEASE_LOG_DISABLED
+    _logIdentifier = presentationModel ? presentationModel->nextChildIdentifier() : nullptr;
```

Comment:
I moved this to ensure `LOGIDENTIFIER` is valid when we call `-setVideoDimensions:` below.